client/layout/masterbar/logged-in.jsx

```diff
@@ -251,7 +251,7 @@ class MasterbarLoggedIn extends Component {
 		this.props.savePreference( MENU_POPOVER_PREFERENCE_KEY, true );
 	};
 
-	renderCheckout() {
+	renderCheckout( showHelpCenter ) {
```

Comment:
Do we need the additional argument here when we could do the changes directly in `renderCheckout()`?